README.md

```diff
@@ -27,10 +27,15 @@ Thanks for your interest in working with Enso. We're currently looking to fill t
   Enso. If you bring strong technical skills and a passion for user experience,
   this could be the role for you.
   [Click here to learn more.](positions/senior-rust-ide-developer.md)
-- **[☁️ Senior Rust Cloud Developer](positions/senior-rust-cloud-developer.md)** to take charge of the design, development and
-  evolution of Enso Cloud. If you have passion for building SaaS products and strong
-  technical skills, this could be the role for you.
+- **[☁️ Senior Rust Cloud Developer](positions/senior-rust-cloud-developer.md)** to take 
+  charge of the design, development and evolution of Enso Cloud. If you have passion for 
+  building SaaS products and strong technical skills, this could be the role for you.
   [Click here to learn more.](positions/senior-rust-cloud-developer.md)
+- **[🤖 Senior Libraries Developer](senior-libs-developer.md)** to design and implement 
+  high-performance data processing libraries for Enso. If you are experienced in implementing 
```

Comment:
We should appeal to passion and interests, not only to experience

Reply:
If you are passionate about functional languages – especially those with immutable table memory model and have experience in implementing libraries in it, this could be the role for you.

how does it sound now?